src/handlers/start/helpers/error-messages.ts

```diff
@@ -0,0 +1,111 @@
+import { Context } from "../../../types/index";
+import { HttpStatusCode, Result } from "../../../types/result-types";
+import { StartEligibilityResult } from "../api/helpers/types";
+
+export const ERROR_MESSAGES = {
+  UNASSIGNED: "{{username}} you were previously unassigned from this task. You cannot be reassigned.",
+  MAX_TASK_LIMIT: "You have reached your max task limit. Please close out some tasks before assigning new ones.",
+  MAX_TASK_LIMIT_PREFIX: "You have reached your max task limit",
+  MAX_TASK_LIMIT_TEAMMATE_PREFIX: "has reached their max task limit",
+  ALL_TEAMMATES_REACHED: "All teammates have reached their max task limit. Please close out some tasks before assigning new ones.",
+  PARENT_ISSUES: "Please select a child issue from the specification checklist to work on. The '/start' command is disabled on parent issues.",
+  CLOSED: "This issue is closed, please choose another.",
+  ALREADY_ASSIGNED: "You are already assigned to this task. Please choose another unassigned task.",
+  ISSUE_ALREADY_ASSIGNED: "This issue is already assigned. Please choose another unassigned task.",
+  PRICE_LABEL_REQUIRED: "You may not start the task because the issue requires a price label. Please ask a maintainer to add pricing.",
+  PRICE_LIMIT_EXCEEDED:
+    "While we appreciate your enthusiasm @{{user}}, the price of this task exceeds your allowed limit. Please choose a task with a price of ${{userAllowedMaxPrice}} or less.",
```

Comment:
I don't recall reading this message before. Perhaps this is incorrect. 

```suggestion
    "The price of this task exceeds your allowed limit. Please choose a task with a price of ${{userAllowedMaxPrice}} or less.",
```